test/e2e/tests/navbar/appearanceMenu.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {expect} from '@playwright/test'
+import {test} from '@sanity/test'
+
+test('color scheme changes and persists', async ({page, baseURL}) => {
+  await page.goto(baseURL ?? '/test/content')
+  await page.locator(`[id='user-menu']`).click()
```

Comment:
❓Does it default store `system` as the value?

If so maybe another expect to check default is system if nothing is selected?

Reply:
done with timeout!